HW_9/BOT/logger.py

```diff
@@ -0,0 +1,8 @@
+import logging
+
+logging.basicConfig(
+    level=logging.INFO,
+    filename="mylog.log",
+    format="%(asctime)s - %(funcName)s: %(lineno)d - %(message)s",
+    datefmt='%H:%M:%S'
```

Comment:
В моей платформе Windows при запуске в логе появляются нечитаемые символы. 
![image](https://user-images.githubusercontent.com/49741194/203966732-fd4cb627-2992-4194-bfbf-b8e7f1e1d294.png)
Поэтому не нужно забывать о кодировке.
```suggestion
    datefmt='%H:%M:%S',
    encoding='utf-8'
```